data-view.js

```diff
@@ -0,0 +1,30 @@
+
+
+async function fetchData() {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/todos/');
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
```

Comment:
If fetch fails will it not trigger the catch by default?

Reply:
done